src/core/ledger/ledger.js

```diff
@@ -43,9 +48,30 @@ type MutableAccount = {|
   // may not receive or transfer Grain. Accounts start inactive, and must
   // be explicitly activated.
   active: boolean,
+  // key-value store of addresses the account receives grain to
+  addresses: PayableAddressStore,
```

Comment:
Could be confused for nodeAddresses or something else. `payoutAddresses` would be a better name